### PR TITLE
Run GPU tests for all PRs.

### PR DESCRIPTION
**Pull request by mindprince**
Fixes kubernetes/kubernetes#47192

### CODE REVIEW

**Comment by mindprince** [comment]
For now, I want this test to run for all PRs but don't want it to be blocking yet (at least for a week or so.)

/cc @vishh

File: prow/config.yaml [hunk 1]
```diff
@@ -187,6 +187,12 @@ presubmits:
     context: pull-kubernetes-e2e-gce-canary
     rerun_command: "/test pull-kubernetes-e2e-gce-canary"
     trigger: "(?m)^/test pull-kubernetes-e2e-gce-canary,?(\\s+|$)"
+  - name: pull-kubernetes-e2e-gce-gpu
+    agent: jenkins
+    always_run: true
```
Comment (krzyzacy) [inline prow/config.yaml]: `skip_report: true` if you don't want to block

**Comment by krzyzacy** [comment]
and add an entry to https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-pull-json.yaml

**Comment by BenTheElder** [comment]
Other than what @krzyzacy already pointed out, this looks good.

**Comment by BenTheElder** [comment]
Additionally for that build failure it looks like you need to:
- run `bazel run //jobs:config_sort`
- also add the job to `kubernetes-security/kubernetes`

**Comment by mindprince** [comment]
Okay, it's passing now.

**Comment by cmluciano** [comment]
For existing PRs, do you need to close/open for these to kick in or is it sufficient to pass the retest command?

**Comment by BenTheElder** [comment]
I think you may need to `/test pull-kubernetes-e2e-gce-gpu` on existing PRs

**Comment by mindprince** [comment]
So, these tests are running for the PRs: https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/pr-logs/directory/pull-kubernetes-e2e-gce-gpu and show up in https://k8s-gubernator.appspot.com/pr/NUM

But this link doesn't show up either in https://k8s-gubernator.appspot.com/ or in the GitHub PR checks list. How do I make it show there?

**Comment by BenTheElder** [comment]
I believe that is only required jobs, and for the last link only some jobs
show up unless you click through to the full list (
https://k8s-gubernator.appspot.com/jobs/kubernetes-jenkins/pr-logs/directory/
).

**Pull request merged**